subsys/logging/Kconfig.backends

```diff
@@ -325,6 +325,41 @@ source "subsys/logging/Kconfig.template.log_format_config"
 
 endif # LOG_BACKEND_ADSP
 
+config LOG_BACKEND_CAVS_HDA
+	bool "cAVS HDA backend"
+	depends on SOC_FAMILY_INTEL_ADSP && DMA && DMA_CAVS_HDA
```

Comment:
Since you only implement the `.process` API, this needs to depend on `CONFIG_LOG2` also.

Reply:
When can we delete v1?

Reply:
Probably after another release? Deprecated stuff needs to stay for 2 releases before they can be deleted.